src/core/radio/trel_interface.cpp

```diff
@@ -352,6 +352,24 @@ void Interface::RemovePeerEntry(Peer &aEntry)
     mPeerTable.PopBack();
 }
 
+void Interface::SendPacket(otInstance       *aInstance,
+                           const uint8_t    *aUdpPayload,
+                           uint16_t          aUdpPayloadLen,
+                           const otSockAddr *aDestSockAddr)
+{
+    otPlatTrelSend(aInstance, aUdpPayload, aUdpPayloadLen, aDestSockAddr); // mCounters
+    ++mCounters.mTxPackets;
+    mCounters.mTxBytes += aUdpPayloadLen;
+    if (error != kErrorNone)
```

Comment:
We dont have any `error` variable in this method?

Reply:
Removed this function as the counters are tracked in platform code.